spi/src/main/java/org/jboss/pnc/spi/environment/RunningEnvironment.java

```diff
@@ -51,4 +52,9 @@
      */
     RepositorySession getRepositorySession();
 
+    /**
+     * @return Returns a build directory.
+     */
+    Path getWorkingDirectory();
```

Comment:
Do we need to care about workingDirectory? Wouldn't be better to work with relative paths?